### ISSUE TITLE
Review the accessibility of the application and fix any issues

### ISSUE THREAD

**Issue by TickleThePanda** [UserOfficeProject/stfc-user-office-project#68]
We need to review the accessibility of any areas of the application we are planning on using. We should document any problems and open issues to fix them.

/overwrite opened 2020-09-04

**Comment by JEThompson97** [comment]
I've done a quick first pass accessibility review using the [WAVE tool](https://github.com/isisbusapps/ISISBusApps/wiki/Accessibility#web-page-accessibility-assessment-tools).
The key _potential*_ issues identified are as follows:
> *I say _potential_ because as well as flagging clear issues, the WAVE tool flags things which _might_ be issues but which need manual checking.

1. **Missing form labels** (Title, Abstract)
1. **Empty buttons** (Button with user icon, Burger menu button)
1. **Skipped heading levels** ('New Proposal' title jumps to h6 on the homepage, and h4 on 'ProsposalCreate')
1. **Contrast too low** (between text and background)
1. **Text too small** ("Privacy Statement", "FAQ", "Drop here!")



Issues 4 and 5 above (contrast and font size) may be addressed as part of #69 (I've created a comment on the issue suggesting this)

**Comment by JEThompson97** [comment]
Panda and I discussed breaking down Accessibility issues between different _Roles_ (User, User Officer, ...).

Along these lines, I will initially work on accessibility issues affecting the _User_ role.

**Comment by JEThompson97** [comment]
Accidently deleted the comment listing the main areas of concern for accessibility.
Here they are again:
1. Contrast too low
2. Text too small
3. Empty buttons
4. Missing labels
5. Skipped header levels
